test-utils/test-runner/Cargo.toml

```diff
@@ -43,7 +43,7 @@ sp-runtime-interface = { version = "3.0.0", path = "../../primitives/runtime-int
 # pallets
 frame-system = { version = "3.0.0", path = "../../frame/system" }
 
-parity-scale-codec = "1.3.1"
+parity-scale-codec = "2.0.0"
 env_logger = "0.7.1"
 log = "0.4.8"
 futures01 = { package = "futures", version = "0.1.29" }
```

Comment:
is it actually used? what about futures01 and rand?

Reply:
Scale-codec is also not used anywhere.

Reply:
rand is used, but not futures01

Reply:
rand in used here
https://github.com/paritytech/substrate/blob/aa3344161522648039fd1080f03396775d1a13bf/test-utils/test-runner/src/node.rs#L160
but is it actually needed?
https://github.com/libp2p/rust-libp2p/blob/bb206bcfdb8f43b71a560cc6938080369b033d48/misc/multiaddr/src/protocol.rs#L80